### PR TITLE
Publish fix, version bump to 2.2.1

### PR DESCRIPTION
**Pull request by dylan-holt-sage**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "rimraf": "^3.0.2",
     "ts-node": "^9.1.1",
     "tslint": "^5.20.1",
-    "typedoc": "^0.23.8",
+    "typedoc": "^0.19.2",
```
Comment (pemba-sherpa-sage) [inline package.json:68]: looks unrelated change

Comment (dylan-holt-sage) [reply package.json:68]: I did it for documentation warning, not a breaking change or anything

**Review by pemba-sherpa-sage — APPROVED** [review]
